### ISSUE TITLE
Fix Canto::UniProt::OntologyAnnotationLookup for non-GO ontologies

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1522]
I think that's the cause of this error after I add a phenotype annotation:

Use of uninitialized value $aspect in hash element at lib/Canto/UniProt/OntologyAnnotationLookup.pm line 119, <GEN51> line 1.

**Comment by ValWood** [comment]
Is this current? I haven't seen it?

**Comment by kimrutherford** [comment]
> Is this current? I haven't seen it?

Yep, it still happens.  I see the message on the console.  I've changed to low priority but I'd still like to fix it.

**Comment by ValWood** [comment]
If this is specific to the uniprot instance
